test-cases/features/compaction-throughput-limit.yaml

```diff
@@ -6,7 +6,7 @@ n_db_nodes: 3
 n_loaders: 1
 n_monitor_nodes: 1
 
-instance_type_db: 'i3.xlarge'
+instance_type_db: 'i4i.xlarge'
```

Comment:
Need to verify that this test doesn't assume specific throughput that is going to be higher with i4i.

Reply:
@fgelcer I'll need help with this.

Reply:
tomorrow once you start working, run this test using `i4i` instance and `i3` instance, so we can compare the results